webapp/components/login/login.jsx

```diff
@@ -35,6 +35,8 @@ export default class Login extends React.Component {
         this.state = state;
     }
     componentDidMount() {
+        document.title = global.window.mm_config.SiteName;
```

Comment:
@crspeller how would that work?  componentDidMount() is only called once since it's the root.  When you navigate to /login it doesn't get re-called.

Or did you mean by some other means?